runners/java-fn-execution/src/test/java/org/apache/beam/runners/fnexecution/InProcessSdkHarness.java

```diff
@@ -98,8 +113,10 @@ public ManagedChannel forDescriptor(ApiServiceDescriptor apiServiceDescriptor) {
           return null;
         });
 
-    client = SdkHarnessClient.usingFnApiClient(clientPool.getSource().get(),
-        dataServer.getService());
+    // HACK: Worker ids cannot currently be set by harness. Everything uses the null id for now.
```

Comment:
<!--thread_id:cc_182896330_t; commit:1fd121da1417624b3b84f0300648251da64b9cb5; resolved:1-->
<!--section:context-quote-->
> **jkff** wrote:
> Is there a JIRA about them being set by the harness?

<!--section:body-->
Just discovered this today and hadn't had a chance to add a jira. Done.